reviews/review-priorities.md

```diff
@@ -0,0 +1,68 @@
+# PR Prioritisation
+
+The Mbed TLS core team prioritise PRs to help us spend our review capacity on the items that best help the project.
+
+When PRs are first triaged, a label is applied to help reviewers determine what to work on (and what not to work on). The different classifications are described below.
+
+Please note that if your PR falls into the lower priority categories, we are happy to increase its priority if needed. This might happen through:
+
+- Commenting on the PR to explain why there is an important use-case for it
+- Other users commenting to show wider demand for the PR
+- Attending the bi-weekly Tech Forum call to discuss the PR
+- Advocating for the PR on the project mailing list
+- Find others to review your PR
+
+Classifications are done through labels, e.g.: priority-high.
+
+Please note that these are the priorities used by the core team. External contributors are welcome to review PRs according to their own priorities, and such review activity will help highlight PRs which should get attention from the core team.
+
+
+### priority-high
+
+This classification is used for PRs which are typically one of:
+
+- Security (or other high-priority) bug-fixes
+- Items that the Mbed TLS team have planned for delivery in the current quarter
+
+These PRs should receive review promptly.
+
+
+### priority-medium
+
+This is for PRs which advance the core team's aims for Mbed TLS, but are not needed to meet delivery in the current quarter. These would typically address one of the following areas:
+
+* Security
+  * Non-urgent and lower-priority security improvements
+* Robustness
+  * Non-urgent bug-fixes
+  * Test improvements
+  * Documentation
+* PSA Crypto
+  * Provide a reference implementation for PSA Crypto
+  * Driver interface for secure elements & crypto processors
+* TLS
+  * Support latest TLS versions and features - especially client features important for embedded devices
+  * DTLS, QUIC
+* Embedded applications
+  * Improvements to code-size, memory footprint and performance on embedded devices
+
```

Comment:
There's no equivalent of the sentence "These PRs should receive review promptly" that the high-priority case has. Would it help to be explicit with something like the following?

`These PRs will receive review on a best-effort basis, but assistance from the community would be helpful`

(or whatever we actually want to say)

Reply:
(note that we have an equivalent for low-priority of `unlikely to receive review soon`, so I think it would be helpful to be explicit as to what our promise is or isn't, rather than just leaving it implicitly somewhere between the high and low guarantees)